### PR TITLE
Update `<Modal>` styles

### PR DESCRIPTION
**Pull request by sashachabin**
- [x] Removed double scrollbar when the `<dialog>` opened
- [x] Added animations
- [x] Added roundings to scrollbar
- [x] Added props for Modal for map
- [x] Updated width/spacers etc.

https://github.com/ekaterinburgdev/transport/assets/22644149/81e459c0-3fea-44cc-8bb5-74b379780e6e

### CODE REVIEW

File: client/components/UI/Modal/Modal.module.css [hunk 1]
```diff
@@ -53,11 +51,21 @@
     height: 18px;
 }
 
-@media screen and (min-width: 768px) {
+@media screen and (min-width: 991px) {
```
Comment (hiba9201) [inline client/components/UI/Modal/Modal.module.css:54]: Interesting breakpoint

File: client/components/UI/Modal/Modal.module.css [hunk 2]
```diff
@@ -68,14 +76,25 @@
...
+    }
+}
+
+@keyframes fadeIn {
+    from {
+        opacity: 0;
+        transform: translateY(-24px) translateZ(0);
+    }
+
+    top {
```
Comment (hiba9201) [inline client/components/UI/Modal/Modal.module.css:96]: top 🔝 
apart from that. do we really need keyframes here? it's for it to start when modal just shows?

File: client/components/UI/Modal/Modal.tsx [hunk 3]
```diff
@@ -3,13 +3,21 @@ import classNames from 'classnames/bind';
...
 import Close from 'public/icons/close.svg';
 
 import styles from './Modal.module.css';
 
 const cn = classNames.bind(styles);
 
-export function Modal({ title = null, children, onClose = () => {} }) {
+export function Modal({
+    title = null,
+    alignCenter = false,
```
Comment (hiba9201) [inline client/components/UI/Modal/Modal.tsx:16]: what about `align: 'center' | 'left' | 'right'`?

File: client/components/UI/Modal/Modal.module.css (hunk 2 above)
Comment (sashachabin) [reply client/components/UI/Modal/Modal.module.css:96]: > top 🔝

fix it

> it's for it to start when modal just shows?

Yep

File: client/components/UI/Modal/Modal.module.css (hunk 1 above)
Comment (sashachabin) [reply client/components/UI/Modal/Modal.module.css]: yep, revert this

**Review by hiba9201 — APPROVED** [review]

**Pull request merged**